src/views/List.jsx

```diff
@@ -25,11 +24,23 @@ export function List({ data }) {
 					onChange={onSearchChange}
 					value={searchField}
 				/>
-				<ul>
-					{filteredLists.map(({ name, id }) => (
-						<ListItem key={id} name={name} />
-					))}
-				</ul>
+
+				{filteredLists && filteredLists.length > 0 ? (
+					<ul>
+						{filteredLists.map(({ name, ...items }) => (
+							<ListItem
+								key={items.id}
+								name={name}
+								items={items}
+								listToken={listToken}
+							/>
+						))}
+					</ul>
+				) : (
+					<div>
+						<p>Loading...</p>
```

Comment:
This is a very nice UX touch, to show the user that the list is still loading :+1: . However, this causes the list to also show "Loading..." when you search for an item that doesn't exist in the list. 
The proper way to show if there are items still loading is to check if the response from firebase is complete and data has been fetched.

Reply:
I was trying this out as it wasn't part of this week's task. I think I will reverse it to what it was initially @faysvas

Reply:
Yeah, I think this was probably part of a previous task